### PR TITLE
Hello Rails React: Project Implementation

### PR DESCRIPTION
**Pull request by rmiyoyo**
# Work Done

 - Created a new Rails app called 'hello-rails-react'.
      - Used the command `rails new -j webpack my_new_rails_app` .
 - Verified that the `jsbundling-rails` gem is added to Gemfile.
 - Postgres database set up. 
 - Created a table for storing messages and created 5 different greetings. 
 - Created an API endpoint that selects a random greeting from the table (controller with an action and Rails route).
 - Created a static view that will be the root of the app.
---------
- Created the `App` component with react-router and rendered it in static view.
- Created the `Greeting` component that displays a greeting. Set it up as a route in the App component.
- Created a store, an action, and a reducer that connects to API endpoint to get the random greeting.
- Displayed the random greeting in the `Greeting` component.

### CODE REVIEW

**Comment by OyePriscilla** [comment]
Hi @rmiyoyo 👍, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have postgres setup as database. 👇 

## Project Requirement

Untitled


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by rmiyoyo** [comment]
# Work Done
- Hey @OyePriscilla, thank you for pointing that out, I have made the changes.

File: db/seeds.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+greetings = ['What is up John', 'Aloha John', 'Habari John']
+
```
Comment (OyePriscilla) [inline db/seeds.rb]: - [ ] Please kindly increase the number of the greeting messages to 5 as stated in the requirement. Note: Don't forget to increase the randomnization also in` /api/greetings_controller.rb`. 🙏 

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @rmiyoyo 👏, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implement a connection between a Ruby on Rails back-end and React front-end. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rmiyoyo** [comment]
# Work Done
- Increased randomization to five

**Review by OyePriscilla — APPROVED** [review]
Hi @rmiyoyo 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Created five random messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**